src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -270,7 +270,7 @@ public Object parse(@NotNull final MinecraftServer server, @NotNull final IComma
             case CITIZEN:
                 return parseCitizenDataValue(parsedHolders, potentialArgumentValue);
             case STRING:
-                return potentialArgumentValue;
+                return potentialArgumentValue.isEmpty() ? null : potentialArgumentValue;
```

Comment:
Either mark this method as Nullable or revert this Change.

Reply:
Yes, that's my fault.   This method is Nullable.  All of the parse*() methods this calls are also Nullable and should be marked as such.

Reply:
I added all of them in my last commit.
I also added a test case.

Btw @mkienenb great job on the command system, it is really easy to use and add to it.